docker/django/docker-entrypoint.sh

```diff
@@ -20,12 +20,20 @@ if [[ "$CREATE_SUPERUSER" = "true" ]]; then
     ./manage.py create_admin_superuser
 fi
 
+if [[ "$COLLECT_STATIC" = "true" ]]; then
+    ./manage.py collectstatic
+fi
+
+if [[ "$INSTALL_TEMPLATES" = "true" ]]; then
+    ./manage.py install_templates helevents
```

Comment:
As other templates are supported, I think it would be nicer if this allowed those to be installed also, not just Helsinki ones. Maybe the installed templates' name could come from the variable?